source/new-starter/prototype/index.html.md.erb

```diff
@@ -89,3 +89,21 @@ If you have set up the apps correctly, you will be able to access the following
 When you have set up your local account, you can [sign into your account](http://www.login.service.dev.gov.uk/) and view the manage screens.
 
 You have now set up and run the GOV.UK account manager prototype, and integrated the `finder-frontend` Brexit transition checker with the GOV.UK account manager.
+
+## Start the account manager prototype without the checker
+
+Some features do not require the checker to be present, and can be tested locally without the checker running.
+These include:
+- Signup (without any Brexit checker data) - /new-account
```

Comment:
I wasn't sure what was clearer here... to just put the path or to put the full URL that you would use if you had things spun up locally?

Reply:
Just to clarify as I'm slightly unclear about this. Am I right in thinking that once you've set up everything up to an including installing `detect-secrets`, you can do one of the following:

- Start the Brexit transition checker and account manager prototype apps
- Start the account manager prototype without the Brexit transition checker

And if you choose the second, then you can go directly to the URLs in this list?

If I'm understanding correctly, then I would make these descriptive links for accessibility.

Reply:
Ace, will amend those as discussed